ricecooker/commands.py

```diff
@@ -298,7 +301,6 @@ def upload_files(tree, file_diff):
         file_diff ([str]): list of files to upload
     Returns: None
     """
-    # Upload new files to CC
```

Comment:
I appreciate the typo correction elsewhere - but let's leave these deleted comments as is!